triplets/arm64-osx-asan.cmake

```diff
@@ -2,6 +2,8 @@ set(VCPKG_TARGET_ARCHITECTURE arm64)
 set(VCPKG_CRT_LINKAGE dynamic)
 set(VCPKG_LIBRARY_LINKAGE static)
 
+
+set(VCPKG_LINKER_FLAGS "-fsanitize=address")
```

Comment:
I'm going to make these changes in another branch for a PR. Thanks for reminding me 👍

Reply:
Opened in #912

Reply:
@2over12 could you remove the changes to this file from this PR please

Reply:
Done. Should be fine with your other commit as long as we squash this PR's commits